modules/foreman_installation_medium.py

```diff
@@ -27,7 +27,8 @@
 author:
   - "Manuel Bonk(@manuelbonk) ATIX AG"
 requirements:
-  - nailgun >= 0.16.0
+  - "nailgun >= 0.16.0"
```

Comment:
I'd be awesome if we could verify this, but I can't really think of a good way to do so.

Reply:
yeah - me neither really.